### PR TITLE
[9.0][MIG] website_sale_require_login: Upgrade to v9

### PR DESCRIPTION
**Pull request by lasley**
This brings website_sale_require_login changes from #89, but with proper rebasing for author attribution. 

There were also some parts that weren't upgraded & some JS Lints, so I handled those as well.

Dependent on merge of:
- [x] #125

### CODE REVIEW

**Comment by pedrobaeza** [comment]
There's an strange merge commit in your history. You can also squash commits from "OCA bot". And please check Travis status.

File: website_sale_require_login/README.rst [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+   :target: https://runbot.odoo-community.org/runbot/113/9.0
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/e-commerce/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed `feedback
+<https://github.com/OCA/
```
Comment (pedrobaeza) [inline website_sale_require_login/README.rst]: Use new template version.

File: website_sale_require_login/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# © 2015 Antiun Ingeniería, S.L. - Jairo Llopis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Require login to checkout",
+    "summary": "Force users to login for buying",
+    "version": "9.0.1.0.0",
+    "category": "Website",
+    "website": "http://www.antiun.com",
+    "author": "Antiun Ingeniería, S.L., "
```
Comment (pedrobaeza) [inline website_sale_require_login/__openerp__.py]: Please switch to the new company `Tecnativa`

**Comment by lasley** [comment]
@pedrobaeza - [This merge (0c08a7428b0a94697f6ebc1fc36939acf1c4cad8)](https://github.com/OCA/e-commerce/pull/124/commits/0c08a7428b0a94697f6ebc1fc36939acf1c4cad8)? Isn't that required to bring the module onto v9 when it is not already present on that branch?

Travis is failing because of lacking #125 as dependency most likely - I accidentally submitted the PRs backwards, with the dependency coming in after the dependent 😆

**Comment by pedrobaeza** [comment]
@lasley, you don't need to make any merge on your migration procedure. Please see https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-9.0 for several migration procedures (we are now trying the experimental one and seems to be easier).

**Comment by lasley** [comment]
@pedrobaeza This module isn't on `9.0`, so there is a `git merge 8.0-extract` in the instructions. Am I missing something? 😖

**Comment by pedrobaeza** [comment]
Please rebase this one

**Comment by lasley** [comment]
Rebased onto 9.0

**Review by pedrobaeza — APPROVED** [review]
Seeing https://github.com/OCA/e-commerce/pull/124/commits/af0595c78371c3c00e0945bfd1a3bbad64f7574f, everything seems correct, except these 2 little comments

**Comment by lasley** [comment]
I'm so happy that inline comments are getting squashed when resolved again. I didn't realize how helpful that was until it was gone.

Changes made btw

**Review by rafaelbn — APPROVED** [review]
Tested in runbot

**Pull request merged**